integration-tests/jest.config.js

```diff
@@ -1,4 +1,16 @@
 module.exports = {
+  verbose: false,
```

Comment:
Jest bug workaround related to `includeConsoleOutput` flag.
https://github.com/jest-community/jest-junit/issues/45#issuecomment-514398747